x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.ts

```diff
@@ -45,7 +45,9 @@ export const rulesNotificationAlertType = ({
     const ruleParams = { ...ruleAlertParams, name: ruleName, id: ruleAlertSavedObject.id };
 
     const fromInMs = parseScheduleDates(
-      previousStartedAt ? previousStartedAt.toISOString() : `now-${ruleParams.interval}`
+      previousStartedAt
+        ? previousStartedAt.toISOString()
+        : `now-${ruleAlertSavedObject.attributes.schedule.interval}`
```

Comment:
NOTE:

I fixed both the typings to be correct and this should have been using:

```ts
ruleAlertSavedObject.attributes.schedule.interval
```

instead of:

```ts
ruleParams.interval
```

Which was causing the exception. Please test this code path.

Reply:
I verified that the interval passed in here is the one that I set during rule creation 👍 . It's not clear to me what the value of `ruleParams.interval` would have been, though, and how that relates to the bug.

Reply:
The bug here was that it was trying to pull the interval from `ruleParams.interval` but that was due to my bad types and kind of the "types hair ball" I introduced which makes it easy to make mistakes.

I fixed the types for this one using `Omit` and then I changed this to use the correct location of the interval which is:

```ts
ruleAlertSavedObject.attributes.schedule.interval
```